File: php/sort/selection-sort/SelectionSortTest.php
<?php

require __DIR__ . '/SelectionSort.php';

class SelectionSortTest
{
    public static function main(): void
    {
        $a = SelectionSort::sort([9, 3, 5, 7, 1]);
        print_r($a);

        $b = SelectionSort::sort([10, 6, 8, 4, 2]);
        print_r($b);
    }
}

SelectionSortTest::main();

File: java/sort/src/main/java/sort/selection/SelectionSortTest.java
package sort.selection;

import java.util.ArrayList;
import java.util.Arrays;
import java.util.List;

import sort.util.SortUtil;

public class SelectionSortTest {

    public static void main(String[] args) {
        List<Integer> a = new ArrayList<>(Arrays.asList(5, 9, 1, 7, 3));
        SortUtil.dump(SelectionSort.sort(a));

        List<Integer> b = new ArrayList<>(Arrays.asList(10, 6, 4, 8, 2));
        SortUtil.dump(SelectionSort.sort(b));
    }

}
